package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "homepage": "/familie/alene-med-barn/soknad",
   "scripts": {
-    "start": "craco start",
+    "start": "craco start --verbose",
     "build": "craco build",
```

Comment:
Hva gjør --verbose?

Reply:
Ah, den kan fjernes. Var bare for å teste noe greier :) Good catch.